packages/mermaid/src/docs.mts

```diff
@@ -131,6 +133,41 @@ const copyTransformedContents = (filename: string, doCopy = false, transformedCo
   logWasOrShouldBeTransformed(fileInFinalDocDir, doCopy);
 };
 
+const transformAnnotation = (content: string, type: 'warning' | 'tip' | 'note') => {
+  let transformed = content;
+  let regex = /::: note\n[\s\S]+?:::/gm;
+  let text = 'Note';
+
+  switch (type) {
+    case 'note':
+      regex = /::: note\s?\n[\s\S]+?:::/gm;
+      break;
+    case 'tip':
+      regex = /::: tip\s?\n[\s\S]+?:::/gm;
+      text = 'Note';
+      break;
```

Comment:
is text for tip `Note`?

Reply:
markdown don't have the equivalente for tip
this is only used to created the docs in the root to be read inside github